provision/docker/provisioner.go

```diff
@@ -40,13 +41,28 @@ func getRouterForApp(app provision.App) (router.Router, error) {
 	return router.Get(routerName)
 }
 
-type dockerProvisioner struct{}
+type dockerProvisioner struct {
+	cluster *cluster.Cluster
+}
+
+func (p *dockerProvisioner) StartupMessage() (string, error) {
+	nodeList, err := p.cluster.UnfilteredNodes()
```

Comment:
One last change, I promise :)
We don't need `p.cluster` here. You can directly call `dockerCluster().UnfilteredNodes()` as `dockerCluster()` already takes care of initializing and using a single cluster instance for the whole package.
With this change you can remove `cluster` attribute from the struct.